musonant/codelab
# Setup react navigation

#### What does this PR do?
* Setup react navigation for the app

#### Description of Task to be completed?
* Configure navigation with react navigation in `App.js`
* Transfer homepage to `HomeScreen.js`

#### How should this be manually tested?
* Clone repository and start app using `yarn start`
* On the home screen, click any of the user thumbnails 
* A new page should open

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories? (if applicable)
NA

#### Screenshots
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/42358974/60011285-5ef38800-9671-11e9-9236-1a0849b85fb8.gif)

![ezgif com-video-to-gif (1)](https://user-images.githubusercontent.com/42358974/60029914-ece36900-9699-11e9-855b-dca4b0a6a1c9.gif)
